.evergreen/config.yml

```diff
@@ -553,6 +553,39 @@ functions:
           KMS_MOCK_SERVERS_RUNNING: "true"
         args: [*task-runner, evg-test-kmip]
 
+  start-kms-failpoint-server:
```

Comment:
This function seems like mostly a duplicate of `start-cse-servers`. Can we use that existing function instead of this one?